src/containers/audio-rewards-page/components/SendInputBody.tsx

```diff
@@ -45,6 +51,7 @@ const messages = {
   addressMalformed: 'Please enter a valid address',
   addressRequired: 'Address is required',
   addressIsSelf: 'You cannot send $AUDIO to your own wallet!',
+  validSPLAddress: 'Please enter a valid Solana (SPL) wallet address',
```

Comment:
do you think this terminology is correct? isn't an SPL wallet the derived token wallet? if you put that in versus the solana account wallet, would that work?

Reply:
I agree with Ray, I don't think this is right bc a valid Solana system account will work. Will post on slack

Reply:
resolved offline